packages/database/src/databases/mongo.ts

```diff
@@ -155,11 +159,14 @@ export class ProsopoDatabase extends AsyncFactory implements Database {
                 solutionTree: parsedDataset.solutionTree,
             }
 
-            await this.tables?.dataset.updateOne(
-                { datasetId: parsedDataset.datasetId },
-                { $set: datasetDoc },
-                { upsert: true }
-            )
+            await this.tables?.dataset
+                .updateOne({ datasetId: parsedDataset.datasetId }, { $set: datasetDoc }, { upsert: true })
+                .catch((err) => {
```

Comment:
you're printing an error but continuing?

Reply:
This looks suspiciously like stuff left in while testing